### PR TITLE
feat: add [GitlabLanguageCount] service

### PR DESCRIPTION
**Pull request by sunny0826**
re https://github.com/badges/shields/issues/8077

### CODE REVIEW

File: services/gitlab/gitlab-languages-count.service.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+      queryParams: { gitlab_url: 'https://gitlab.com' },
+      staticPreview: {
+        label: 'languages',
+        message: '5',
+      },
+      documentation,
+    },
+  ]
+
+  static defaultBadgeData = { label: 'languages', namedLogo: 'gitlab' }
```
Comment (calebcartwright) [inline services/gitlab/gitlab-languages-count.service.js:41]: We shouldn't have a default logo on this one, so let's drop it:

```suggestion
  static defaultBadgeData = { label: 'languages' }
```

File: services/gitlab/gitlab-languages-count.service.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+  static render({ languagesCount }) {
+    return {
+      label: 'languages',
```
Comment (calebcartwright) [inline services/gitlab/gitlab-languages-count.service.js:45]: The label is already set in the defaults so we don't need to specify this again

```suggestion
```

**Review by calebcartwright — COMMENTED** [review]
Thanks! Couple minor items inline below

File: services/gitlab/gitlab-languages-count.service.js (hunk 1 above)
Comment (sunny0826) [reply services/gitlab/gitlab-languages-count.service.js:41]: done

File: services/gitlab/gitlab-languages-count.service.js (hunk 2 above)
Comment (sunny0826) [reply services/gitlab/gitlab-languages-count.service.js:45]: done

**Review by calebcartwright — APPROVED** [review]
LGTM, thanks!

**Pull request merged**